music21/mei/base.py

```diff
@@ -1272,63 +1272,67 @@ def makeMetadata(documentRoot):
     work = documentRoot.find(f'.//{MEI_NS}work')
     if work is not None:
         # title, subtitle, and movement name
-        meta = metaSetTitles(work, meta)
-        # composers, etc
-        meta = metaSetContributors(work, meta)
+        meta = metaSetTitle(work, meta)
+        # composer
+        meta = metaSetComposer(work, meta)
         # date
         meta = metaSetDate(work, meta)
 
     return meta
 
 
-def metaSetTitles(work, meta):
+def metaSetTitle(work, meta):
     '''
-    From a <work> element, find the titles, subtitles, and movement name (<tempo> element) and store
+    From a <work> element, find the title, subtitle, and movement name (<tempo> element) and store
     the values in a :class:`Metadata` object.
 
     :param work: A <work> :class:`~xml.etree.ElementTree.Element` with metadata you want to find.
     :param meta: The :class:`~music21.metadata.Metadata` object in which to store the metadata.
     :return: The ``meta`` argument, having relevant metadata added.
     '''
-    # titles, subtitles, and movement name
+    # title, subtitle, and movement name
+    subtitle = None
     for title in work.findall(f'./{MEI_NS}titleStmt/{MEI_NS}title'):
-        if title.get('type', '') == 'subtitle':
-            meta.add('subtitle', title.text)
-        else:
-            meta.add('title', title.text)
+        if title.get('type', '') == 'subtitle':  # or 'subordinate', right?
+            subtitle = title.text
+        elif meta.title is None:
+            meta.title = title.text
+
+    if subtitle:
+        # Since m21.Metadata doesn't actually have a "subtitle" attribute, we'll put the subtitle
+        # in the title
+        meta.title = f'{meta.title} ({subtitle})'
```

Comment:
Can you make an issue to add a subtitle attribute after this is done -- I think it's a valuable field addition.  (Or at least look at how MARC/DC deals with these).

Reply:
Neither MARC nor DC has a subtitle.  But yeah, I'll make an issue.

Reply:
Issue has been written.